src/app/components/settings/settings.html

```diff
@@ -1,5 +1,5 @@
 <div class="settings-header">
-    <h1 tabindex="0" autofocus>{{'settings.title' | i18n}}</h1>
+    <h1>{{'settings.title' | i18n}}</h1>
```

Comment:
It looks like we have this same behavior in auth-settings.html. I'm guessing it was just copy/pasted from settings.html. Can we change that in this PR as well?